Sprint9-JS-Oject/tema2.js

```diff
@@ -0,0 +1,19 @@
+// Scrieți o funcție care returnează un obiect nou fără proprietățile furnizate
+
+function getPartOfObject(obj, p){
```

Comment:
(optional) alege un alt nume, mai sugestiv, in loc de "p". De exemplu "propertyToRemove"

Reply:
Done